test/terraform_aws_ecs_cluster_test.go

```diff
@@ -0,0 +1,57 @@
+package test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	test_structure "github.com/gruntwork-io/terratest/modules/test-structure"
+
+	awsSDK "github.com/aws/aws-sdk-go/aws"
+	"github.com/stretchr/testify/assert"
+)
+
+// An example of how to test the Terraform module in examples/terraform-aws-ecs-example using Terratest.
+func TestTerraformAwsEcsExample(t *testing.T) {
+	t.Parallel()
+
+	tempTestFolder := test_structure.CopyTerraformFolderToTemp(t, "../", "examples/simple")
+
+	expectedClusterName := fmt.Sprintf("terratest-aws-ecs-example-cluster-%s", random.UniqueId())
+	moduleClusterName := expectedClusterName + "-test"
+
+	// Pick a random AWS region to test in. This helps ensure your code works in all regions.
+	awsRegion := aws.GetRandomStableRegion(t, []string{"us-east-1", "eu-west-1"}, nil)
```

Comment:
We've started to move away from the random region functionality in Terratest because it wasn't really showing much value , for example some regions only have 1 AZ.  Another reason is it will be easier to automate cleaning partially failed tests if we localize it to 1 region. We've been using us-west-2.